test/jasmine/assets/click.js

```diff
@@ -4,4 +4,4 @@ module.exports = function click(x, y) {
     mouseEvent('mousemove', x, y);
     mouseEvent('mousedown', x, y);
     mouseEvent('mouseup', x, y);
-};
\ No newline at end of file
+};
```

Comment:
I was thinking it may be worth moving all our `.eslintrc` in a `config/` at the root of the repo instead of having them scattered around (and hidden in) the directory tree.

We could move `test_syntax.js` to `tasks/`. 

Having

```
test/
  |
  - image/
  - jasmine/
  - test_syntax.js
  - .eslintrc  (which is only relevant for test_syntax.js)
```

isn't great.